cmd/scollector/collectors/iterable.go

```diff
@@ -2,68 +2,92 @@ package collectors
 
 import (
 	"context"
-	"fmt"
 	"time"
 
 	"bosun.org/cmd/scollector/conf"
 	"bosun.org/metadata"
 	"bosun.org/opentsdb"
+	"bosun.org/slog"
 	"github.com/bosun-monitor/statusio"
 )
 
+// iterable.io collector
+// to use add this in your scollector.toml:
+// [Iterable]
```

Comment:
We would be better open source citizens if the component keys were selectable from the config. 

If this would mean writing a bunch of metadata that isn't useful for us right now that doesn't mean we have to expand the scope that much. We could only support the ones we care about to merge this, and then if people want to add ones down the road